soak/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.8-alpine
+
+# Persist build arguments into environment variables
+ARG AWS_SERVICE
+ARG E2E_GIT_REF=main
+ARG CONTROLLER_E2E_PATH=./${AWS_SERVICE}-controller/test/e2e
+ENV AWS_SERVICE ${AWS_SERVICE}
+ENV PYTHONPATH ${CONTROLLER_E2E_PATH}
+ENV CONTROLLER_E2E_PATH ${CONTROLLER_E2E_PATH}
+
+WORKDIR /soak
+# Install dependencies for soak test environment
+RUN apk add -y --no-cache git bash
+
+# Install python dependencies for running ack e2e tests
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
As I stated below in another comment, the `requirements.txt` dependencies in the `test-infra` repository are already included as part of the test dependency graph of the service-controller e2e tests. We actually don't want to install them here, because the service-controller e2e tests may be referencing an older version of the `test-infra` `acktest` package.

Might I suggest that after we clone (on L26), we `pip install` the requirements there. This way the container does not need to install requirements at runtime i.e in the `run_soak_test.sh` script.